frontend/apps/vft-extended/README.md

```diff
@@ -0,0 +1,48 @@
+# Example Frontend for Fungible Token
+
+This project demonstrates a minimal frontend for a Fungible Token (VFT) contract using [sails-js](https://wiki.gear.foundation/docs/sails-js/react-hooks) React hooks.
+
+## Key Components
+
+- **API Hooks:** Modularized contract logic, all reusable hooks are in `hooks/api.ts`.
+- **libts:** Auto-generated contract interface description (TypeScript), generated using `sails-js-cli` (see below).
+- **Home:** The main view component showcasing token operations, transfer, and balance queries.
+
+## How to generate `lib.ts`
+
+Generate TypeScript interface files for your contract using the CLI:
+
+```bash
+npm install -g sails-js-cli
+
+sails-js generate path/to/sails.idl -o path/to/out/dir --no-project
+```
+
+More details: [SailsJS Client Generation Guide](https://wiki.gear.foundation/docs/sails-js/client-generation)
```

Comment:
This part can be confusing, as there's no need to generate lib to launch an app.